website/pages/docs/k8s/connect/health.mdx

```diff
@@ -0,0 +1,47 @@
+---
+layout: docs
+page_title: Kubernetes Health Checks
+sidebar_title: Health Checks
+description: Configuring Kubernetes Health Checks
+---
+
+# Kubernetes Health Checks in Consul on Kubernetes
+
+-> 0.25+ This feature is available in consul-helm versions 0.25 and higher
+
+~> This topic requires familiarity with [Kubernetes Health Checks](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/).
+
+This page describes how to enable Kubernetes Health checks to extend Consul for service mesh uses cases.
+
+When deploying your helm chart, you must provide helm with a custom yaml file that contains your environment configuration.
+Enabling health checks is done via the `healthChecks` stanza under `connectInject`.
+
+When `enabled: true` is set, a [TTL health check](https://www.consul.io/docs/discovery/checks) will be registered within
+Consul for each Kubernetes pod that is connect-injected. The Consul health check's state will reflect the pod's readiness
+status, which is the combination of all probes registered with the pod. Kubernetes Readiness probes are the primary use
+case as they can change state without restarting the containers in the pod.
```

Comment:
nit: I was a bit confused by this sentence. Might be worth rephrasing this a bit. Are you trying to explain why we're using readiness checks as opposed to other kube checks?